packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3216,10 +3216,16 @@ export class ContainerRuntime
 			"Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod",
 		);
 		if (recoveryMethod === "restart") {
-			this._summarizer?.stop("latestSummaryStateStale");
-			// TODO: stop execution, but don't cause a cascading series of errors.
 			const error = new GenericError("Restarting summarizer instead of refreshing");
-			this.closeFn(error);
+			this.mc.logger.sendTelemetryEvent(
```

Comment:
do we need this? Calling this.closeFn with the error will log an event anyway right?

Reply:
If logging it, can you add versionId and the props from event that are passed in so we have more details. The passed in eventName can be logged in as something else.

Reply:
Calling close will create an error event in telemetry.

Reply:
that will happen anyway right because close is called. is that error telemetry not enough?

Reply:
This avoids the container close as showing up as error telemetry. That way I think we only get 1 error telemetry instead of 2

Reply:
That makes sense. I did not know that not passing an error to close would log a generic event instead of error.